.github/workflows/build-test-sources.yaml

```diff
@@ -66,16 +66,10 @@ jobs:
       with:
         submodules: recursive
 
-    - name: Install Rust
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: ${{ matrix.plan.rust }}
-        override: true
-        components: rustfmt
-
     - name: Run rustfmt
       run: |
+        rustup default ${{ matrix.plan.rust }}
+        rustup component add rustfmt
```

Comment:
The lines below, e.g. `cargo fmt --manifest-path rust-bins/Cargo.toml -- --check` can be written via a matrix in my opinion.

Reply:
Yes.